src/isSentinelValue.js

```diff
@@ -0,0 +1,26 @@
+import { curryN } from 'ramda';
+
+import isInteger32 from './isInteger32';
+
+/**
+ * Checks whether the passed value is a sentinel value.
+ *
+ * @func isSentinelValue
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.33.0|v2.33.0}
+ * @category Type
+ * @sig * -> Boolean
+ * @param {*} val The value to test
+ * @return {boolean}
+ * @see {@link RA.isSentinelValue|isSentinelValue}
```

Comment:
```suggestion
 * @see {@link RA.isSentinelValue|isSentinelValue}
```
No need to mention the reference to function to itself.

Reply:
Removed the reference